enterprise/frontend/src/embedding-sdk/hooks/private/use-sdk-usage-problem.ts

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useMemo, useRef } from "react";
+import { useDispatch } from "react-redux";
+
+import type { SDKConfig } from "embedding-sdk";
+import { printUsageProblemToConsole } from "embedding-sdk/lib/print-usage-problem";
+import { getSdkUsageProblem } from "embedding-sdk/lib/usage-problem";
+import { setUsageProblem } from "embedding-sdk/store/reducer";
+import { useSetting } from "metabase/common/hooks";
+import { useSelector } from "metabase/lib/redux";
+import { getApplicationName } from "metabase/selectors/whitelabel";
+import { getTokenFeature } from "metabase/setup/selectors";
+
+export function useSdkUsageProblem(config: SDKConfig) {
+  const { allowConsoleLog = true } = config;
+
+  const hasLoggedRef = useRef(false);
+  const appName = useSelector(getApplicationName);
```

Comment:
I'm not sure where we have specced this. This is weird.

https://github.com/metabase/metabase/blob/54825a17312ee32702ea596fd20e3bdce3fee878/enterprise/frontend/src/embedding-sdk/lib/print-usage-problem.ts#L24-L27

Imagine seeing `My super awesome app Embedding SDK for React. Please see the doc` and the doc still links to Metabase.

I really think here we have to say `Metabase Embedding SDK for React`

Reply:
Fixed in https://github.com/metabase/metabase/pull/47821/commits/4aa503024e958329401a25c7776a30cf617895b1, thanks! The `appName` was part of the existing log, but it indeed doesn't make sense.